docs/precommit.md

```diff
@@ -7,28 +7,15 @@ You can use Prettier with a pre-commit tool. This can re-format your files that
 
 ## Option 1. [lint-staged](https://github.com/okonet/lint-staged)
 
-**Use Case:** Useful for when you need to use other tools on top of Prettier (e.g. ESLint)
+**Use Case:** Useful for when you want to use other code quality tools along with Prettier (e.g. ESLint, Stylelint, etc.). Supports partially staged files (`git add --patch`)
 
-Install it along with [husky](https://github.com/typicode/husky):
+_Make sure Prettier is installed and is in your `devDependencies` before you proceed._
 
 ```bash
-yarn add lint-staged husky --dev
+npx mrm lint-staged
 ```
 
-and add this config to your `package.json`:
-
-```json
-{
-  "husky": {
-    "hooks": {
-      "pre-commit": "lint-staged"
-    }
-  },
-  "lint-staged": {
-    "*.{js,json,css,md}": ["prettier --write", "git add"]
-  }
-}
-```
+This will install and configure [husky](https://github.com/typicode/husky) and [lint-staged](https://github.com/okonet/lint-staged) to reformat supported formats on pre-commit hook.
```

Comment:
Shall we mention that the `package.json` will be updated with the necessary rules and if users needs to support more files they should edit there? (Something among those lines, that the pre-commit hooks are done in package.json and not under the .git directory for example)

Reply:
Updated the wording. WYT?